apps/backend/src/user/user.controller.ts

```diff
@@ -1,8 +1,24 @@
-import { Controller } from '@nestjs/common';
+import { Body, Controller, Get, Param, Patch, Query } from '@nestjs/common';
 
+import { UpdateUserDto } from './dto/update-user.dto';
 import { UserService } from './user.service';
 
-@Controller('user')
+@Controller('users')
 export class UserController {
   constructor(private readonly userService: UserService) {}
+
+  @Get()
+  async findAll(@Query('page') page: number = 1, @Query('pageSize') pageSize: number = 10) {
+    return this.userService.findMany(page, pageSize);
+  }
```

Comment:
you have to parse the query values (maybe with parseintpipe) or they will remain strings
![image](https://github.com/user-attachments/assets/23424791-4933-49a1-9b46-94bbfe8c8082)